src/Symfony/Component/Messenger/Stamp/SentStamp.php

```diff
@@ -20,7 +20,7 @@
  *
  * @experimental in 4.3
  */
-final class SentStamp implements StampInterface
+final class SentStamp implements NonSendableStampInterface
```

Comment:
Are any of these useful in seeing the history of the message across retries? Or does RedeliverStamp cover that?

Reply:
RedeliverStamp does that using `getSenderClassOrAlias`